### PR TITLE
fix(DB/creature,SAI): Silvermane Stalkers: grey -> black, no fade

### LINKED ISSUE #5847: Silvermane Stalkers in Hinterlands should not stealth and appear black instead of grey
Original report: https://github.com/chromiecraft/chromiecraft/issues/645 by @ndnd123

**As of patch 3.2.0 they should be black instead of grey and no longer stealth (see below).**

#### WHAT CLIENT DO YOU PLAY ON?
- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION
- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:
- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [x] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:
They do not break stealth when they hit you. If you are facing them they look see-through, if you turn your back on them they are completely invisible.

##### EXPECTED BLIZZLIKE BEHAVIOUR:
They probably should break stealth when they start attacking you.

**Additional information by @necropola**

According to **Wowpedia** from November 2010 they should be black instead of grey and no longer stealth since patch 3.2.0: https://wowpedia.fandom.com/wiki/Silvermane_Stalker?oldid=2482355

And here are **Wowhead** screenshot captions and comments retrieved via **Wayaback Machine** from October 2010:: http://web.archive.org/web/20101030160706/http://www.wowhead.com:80/npc=2926
```JavaScript
var lv_screenshots = [
{id:49234,user:'erk070',date:'2007/09/03 01:11:09',caption:'Original graphics',width:972,height:803,type:1,typeId:2926},
{id:175234,user:'JadedOne',date:'2010/07/21 12:56:58',caption:'Now has black fur and doesn\'t stealth as of patch 3.3',width:177,height:151,type:1,typeId:2926,sticky:1}];


var lv_comments = [
{number:0,id:48790,user:'thatguy',
body:'This NPC cloaks itself, so be careful when your traveling around its main area, especially if you are less than 40',
date:'2007/03/09 16:04:20',rating:5,indent:0,roles:0,purged:0,deleted:0,
raters:[[118,1],[10155,1],[38873,1],[51548,1],[61423,1],[95064,1],[120034,-1],[129598,-1],[144176,-1],[159390,1],[166871,1]],replyTo:48790},

{number:1,id:182333,user:'PDminion',
body:'Unfortunately, no.\n\nWolves can never learn to stealth, if you want a pet able to do that you have to go with a cat.\n\nTip: If you click the (Wolf) link in quick facts, you can see which spells wolves can learn.',
date:'2007/11/11 09:49:21',rating:1,indent:0,roles:0,purged:0,deleted:0,
raters:[[83127,1],[95064,1],[129598,-1],[144176,-1],[154011,-1],[166871,1],[177775,1]],lastEdit:['2007/11/11 09:49:36',1,'PDminion'],replyTo:182333},

{number:2,id:548337,user:'ivenin',body:'[small]...maybe there was a comment before yours that is missing?[/small]\n\nBut these NPC wolves definitely do [url=http://www.wowhead.com/?spell=8218]Sneak[/url]. But as tamed, they are not able to learn any sort of "prowling" or "stealth" spell.',
date:'2009/01/05 14:49:31',rating:2,indent:1,roles:0,purged:0,deleted:0,
raters:[[129598,-1],[144176,-1],[154011,1],[166871,1],[274087,1],[298724,1]],replyTo:182333},

{number:3,id:601264,user:'travellingbard',
body:'The post was most likely referencing someone who made a comment about hunter pets and wolf stealthing.\n\nAlso please note the locations marked on map. This particular mob will be the only one that drops the item you need.'
,date:'2009/02/09 03:26:37',rating:0,indent:1,roles:0,purged:0,deleted:0,
raters:[[144176,-1],[166871,1],[296580,1],[298724,-1]],replyTo:182333},

{number:4,id:798715,user:'Chiasa',body:'As of 3.2, these mobs now have the dark grey worg skin and are no longer stealthed.',
date:'2009/08/05 04:24:23',rating:19,indent:0,roles:0,purged:0,deleted:0,
raters:[[2534,1],[5027,1],[31262,1],[40602,1],[44211,1],[44653,1],[49236,1],[69255,1],[120034,-1],[122553,1],[129598,1],[139412,1],[149270,1],[166871,1],[169698,1],[171419,1],[202833,1],[226705,1],[295419,1],[312910,1],[427375,1]],replyTo:798715},

{number:5,id:817142,user:'yiNXs',
body:'The area where they can be found seems to have changed too. Like the map (still) shows at this moment, they use to fill the whole east part of the map from X=51 till the cliffs at the shore. Now their area seems to start at roughly X=65.',
date:'2009/08/19 21:04:52',rating:8,indent:1,roles:0,purged:0,deleted:0,
raters:[[44211,1],[44653,1],[61423,1],[120034,-1],[166871,1],[171419,1],[202833,1],[226705,1],[312910,1],[337263,1]],replyTo:798715},

{number:6,id:826495,user:'temu92',
body:'Not so "silvermaneish" anymore :P',
date:'2009/08/29 16:59:11',rating:17,indent:0,roles:0,purged:0,deleted:0,
raters:[[4678,1],[5009,1],[40602,1],[44211,1],[55377,1],[61423,1],[69255,1],[77986,1],[100371,-1],[120034,1],[149270,1],[166871,1],[171419,1],[174397,1],[219996,1],[238644,1],[295419,1],[427375,1],[458683,1]],replyTo:826495},

{number:7,id:826707,user:'skeletwing',
body:'Or "stalkerish".',
date:'2009/08/29 22:36:59',rating:19,indent:1,roles:0,purged:0,deleted:0,
raters:[[4678,1],[5009,1],[40602,1],[44211,1],[44653,1],[55377,1],[61423,1],[69255,1],[77986,1],[100371,-1],[149270,1],[165298,1],[166871,1],[171419,1],[174397,1],[219996,1],[238644,1],[295419,1],[344808,1],[427375,1],[458683,1]],replyTo:826495}];
```

##### STEPS TO REPRODUCE THE PROBLEM:
1.  Go to Silvermane Stalker in Hinterlands
2.  Facepull them to get them to attack you
3. Turn your back on them
4. They attack you (and even daze you) while remaining stealthed

##### EXTRA NOTES:
image

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/8dd58ebb5bea6a8aada54c830d06974d3ecd1353

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98542810-silvermane-stalkers-in-hinterlands-should-not-stealth-and-appear-black-instead-of-grey?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by necropola**
Co-authored-by: @Branel

## Changes Proposed:
- Change the spawned model for **Silvermane Stalkers (2926)** from **grey (11418)** to **black (9562)** which matches the template
- Remove **fade** ability from SmartAI
- Unhide nameplate

## Issues Addressed:
- Closes #5847
- Closes https://github.com/chromiecraft/chromiecraft/issues/645

## SOURCE:
See #5847

## Tests Performed:
- SQL statements executed
- Tested in game

## How to Test the Changes:
- Execute the SQL statements from the patch
- Restart your `worldserver`
- Go to **Hinterlands**
- Check out the **Silvermane Stalkers** near **Skulk Rock**. They should be black now, nameplates should be visible without targeting them and they should no longer be stealthed/faded.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]

**Comment by necropola** [comment]
I've noticed that the **Silverman Stalkers** still have a hidden nameplate (unless targeted), but I cannot see why. Pretty similar to #5365. Any ideas?

**Comment by FrancescoBorzi** [comment]
@Helias @Kitzunu don't forget to apply the content-level label

File: data/sql/updates/pending_db_world/rev_1621105426491747047.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621105426491747047');
+
+-- Remove id=1: Silvermane Stalker - On Reset - Cast Faded (6408) at 100% chance (No Repeat)
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2926 AND `id`=1;
+
```
Comment (Branel) [inline data/sql/updates/pending_db_world/rev_1621105426491747047.sql:5]: ```  UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 2926);```  

Add this to resolve missing nameplates

**Review by Branel — COMMENTED** [review]
```  UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 2926);```  

Add this to resolve missing nameplates

File: data/sql/updates/pending_db_world/rev_1621105426491747047.sql (hunk 1 above)
Comment (necropola) [reply data/sql/updates/pending_db_world/rev_1621105426491747047.sql:5]: Thank you!
Now the nameplate is visible.

**Comment by necropola** [comment]
> Add this to resolve missing nameplates

Huge thanks! Added to PR.

**Pull request merged**